composer.json

```diff
@@ -32,7 +32,7 @@
         "php": ">=5.6.3",
         "ext-sockets": "*",
         "ext-mbstring": "*",
-        "phpseclib/phpseclib": "^2.0.0"
+        "phpseclib/phpseclib": "*"
```

Comment:
Asterix allows any version, even lower than 2.0. This can lead to unexpected runtime exceptions.